src/lib/shared_utils/src/canister_specific/individual_user_template/types/hot_or_not/mod.rs

```diff
@@ -688,18 +495,22 @@ impl Post {
 
                 room_details_map.insert(global_room_id, room_detail);
                 if global_room_id.2 != ongoing_room {
-                slot_details_map.insert(
-                    (self.id, ongoing_slot),
-                    SlotDetailsV1 {
-                        active_room_id: global_room_id.2,
-                    },
-                );
+                    slot_details_map.insert(
+                        (self.id, ongoing_slot),
+                        SlotDetailsV1 {
+                            active_room_id: global_room_id.2,
+                        },
+                    );
                 }
 
                 self.hot_or_not_details = Some(hot_or_not_details);
 
                 let started_at = self.created_at;
-                let number_of_participants = (num_bets_made + 1) as u8;
+                let number_of_participants = if num_bets_made >= 100 {
+                    ((num_bets_made + 1) % 100) as u8
+                } else {
+                    (num_bets_made + 1) as u8
+                } as u8;
```

Comment:
is this to prevent overflow ?

Reply:
can just be ((num_bets_made + 1) % 100) as u8 in that case

Reply:
for 99th user it will come 0. we want it to be 100